.gitignore

```diff
@@ -1,2 +1 @@
-node_modules
-./js/lib
+scripts/go/bin
```

Comment:
I'm guessing this isn't needed anymore. If so, maybe to move tool configs into the root of the repo, or maybe only one dir deep?